e2e/app/avs.go

```diff
@@ -48,7 +48,12 @@ func deployAVS(ctx context.Context, def Definition) error {
 
 	addr, receipt, err := avs.DeployIfNeeded(ctx, def.Testnet.Network, backend)
 	if err != nil {
-		return errors.Wrap(err, "deploy")
+		if def.Manifest.Network.IsEphemeral() {
```

Comment:
@kevinhalliday I've been getting `INITILIZATION_FAILED` errors when create3 deploys the avs contract to anvil. Everything else works. Just disabled this for now. We had this in the past, simple retry solved it then.